local_server/qwen3_tts_server/local_server.py

```
@@ -0,0 +1,261 @@
+                if torch.cuda.is_available():
+                    torch.cuda.empty_cache()
+
+        async def _sender_loop():
+            while True:
+                chunk = await audio_queue.get()
+                if chunk == b"__END__":
+                    await websocket.send(json.dumps({"type": "response.done", "job_id": current_job_id}))
+                    continue
+                try:
+                    await websocket.send(chunk)
+                except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)